<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFIND</class>
 <widget class="QWidget" name="CFIND">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Addr_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>420</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>100</y>
     <width>271</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string>记录时间</string>
   </property>
   <widget class="QRadioButton" name="QrbtnTime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>47</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>开始时间：</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="QdateSTime">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>60</y>
      <width>154</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>结束时间：</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="QdateETime">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>90</y>
      <width>154</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="QpbtnFind">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>290</y>
     <width>96</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QpbtnClose">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>290</y>
     <width>96</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="QchkFind">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>60</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="QlineFind">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
